content/hardware/04.pro/boards/portenta-c33/datasheets/datasheet.md

```diff
@@ -1,6 +1,6 @@
 ---
 identifier: ABX00074
-title: Arduino® Portenta C33

```

Comment:
Please leave the Arduino here 🙏🏼 this is consistent with other datasheets

Reply:
Returning original tittle